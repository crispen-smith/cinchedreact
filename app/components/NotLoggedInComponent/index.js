/**
 *
 * NotLoggedInComponent
 *
 */

import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import Not from '../Not';

/* eslint-disable react/prefer-stateless-function */
class NotLoggedInComponent extends React.PureComponent {
  render() {
    return (
      <Fragment>
        <h1>{this.props.title}</h1>
        <Not>You must be logged in to access this page.</Not>
      </Fragment>
    );
  }
}

NotLoggedInComponent.propTypes = {
  title: PropTypes.string.isRequired,
};

export default NotLoggedInComponent;
