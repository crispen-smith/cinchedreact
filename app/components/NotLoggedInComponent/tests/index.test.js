import React from 'react';
import renderer from 'react-test-renderer';

import NotLoggedInComponent from '../index';

describe('<NotLoggedInComponent />', () => {
  it('Renders a BrandBox', () => {
    const tree = renderer
      .create(<NotLoggedInComponent title="Not Logged In Test" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
