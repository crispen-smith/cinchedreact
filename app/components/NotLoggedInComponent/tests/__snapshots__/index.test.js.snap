// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotLoggedInComponent /> Renders a BrandBox 1`] = `
Array [
  <h1>
    Not Logged In Test
  </h1>,
  <div
    className="Not-bJJsFZ hKjuvT"
  >
    You must be logged in to access this page.
  </div>,
]
`;
